frontend/src/components/shared/ProgressBar/ProgressBar.tsx

```diff
@@ -18,31 +18,42 @@
 import React, { ReactElement } from "react"
 import { SCSS_VARS } from "autogen/scssVariables"
 import { ProgressBar as UIProgressBar } from "baseui/progress-bar"
+import { mergeOverrides } from "baseui"
+import { Sizes } from "lib/widgetTheme"
 
 export interface ProgressBarProps {
   width?: number
   value: number
+  overrides?: any
```

Comment:
I believe you should be able to pull `overrides` type definition from `baseui`. Though, I might be wrong.